RadixWallet/Features/AccountDetailsFeature/Coordinator/AccountDetails+Reducer.swift

```diff
@@ -201,43 +137,13 @@ public struct AccountDetails: Sendable, FeatureReducer {
 			state.destination = nil
 			return .none
 
-		case .assets(.child(.fungibleTokenList(.delegate))):
-			return .none
-
-		case let .destination(.presented(.exportMnemonic(.delegate(.doneViewing(markedMnemonicAsBackedUp))))):
-			if
-				let justBackedUp = markedMnemonicAsBackedUp,
-				state.exportMnemonicPrompt.needed,
-				justBackedUp
-			{
-				state.exportMnemonicPrompt = .no
-			}
-			state.destination = nil
-			return checkAccountSecurityPromptStatus(state: &state)
-
-		case let .destination(.presented(.importMnemonics(.delegate(delegateAction)))):
-			switch delegateAction {
-			case .closeButtonTapped, .failedToImportAllRequiredMnemonics, .finishedImportingMnemonics:
-				state.destination = nil
-				return checkAccountSecurityPromptStatus(state: &state)
-			case let .importedMnemonic(factorSourceID):
-				guard factorSourceID == state.deviceControlledFactorInstance.factorSourceID.embed() else {
-					return .none
-				}
-
-				state.importMnemonicPrompt = .no
-				// It makes no sense to prompt user to back up a mnemonic she *just* imported.
-				state.exportMnemonicPrompt = .no
-				return .send(.delegate(.importedMnemonic(factorSourceID)))
-			}
+		case .assets(.delegate(.resourcesUpdated)):
+			return checkIfShouldShowExportMnemonicPrompt(state: &state)
```

Comment:
we can piggyback on the child reducer `assets` here, no need to fetch portfolio here too!